### ISSUE TITLE
Suggest moving from AWS to github pages

### ISSUE THREAD

**Issue by edmundsj** [edmundsj/learnElectricalEngineering#2]
Github pages is free and would make it easier to integrate others changes and provides tight integration with github actions. Would also make it easier from a maintenance standpoint. Would first need to move from PHP to plain HTML / reST / markdown.

**Comment by ghost** [comment]
I agree. Github pages is easier to work with, and free is almost always better than paid!
